ql/src/test/results/clientpositive/perf/tez/constraints/query25.q.out

```diff
@@ -176,92 +176,88 @@ Stage-0
                                   Merge Join Operator [MERGEJOIN_246] (rows=21091882 width=154)
                                     Conds:RS_25._col2, _col1, _col4=RS_26._col2, _col1, _col3(Inner),Output:["_col1","_col3","_col5","_col8","_col9","_col11"]
                                   <-Reducer 10 [SIMPLE_EDGE]
+                                    SHUFFLE [RS_26]
+                                      PartitionCols:_col2, _col1, _col3
+                                      Merge Join Operator [MERGEJOIN_245] (rows=9402909 width=100)
+                                        Conds:RS_270._col0=RS_256._col0(Inner),Output:["_col1","_col2","_col3","_col4"]
+                                      <-Map 8 [SIMPLE_EDGE] vectorized
+                                        PARTITION_ONLY_SHUFFLE [RS_256]
+                                          PartitionCols:_col0
+                                          Select Operator [SEL_252] (rows=351 width=4)
+                                            Output:["_col0"]
+                                            Filter Operator [FIL_250] (rows=351 width=12)
+                                              predicate:((d_year = 2000) and d_moy BETWEEN 4 AND 10)
+                                              TableScan [TS_3] (rows=73049 width=12)
+                                                default@date_dim,d3,Tbl:COMPLETE,Col:COMPLETE,Output:["d_date_sk","d_year","d_moy"]
```

Comment:
Are these new TS that are not reused anymore? For instance, it seems this one is the same that was reused in old L224.

Reply:
date_dim is only scanned once in the new plan as well - in this query the order of things have changed a bit
so far it seems like the biggest change is that `Reducer 10` is exchanged with `Reducer 15`